lib/mongoose_push_web/endpoint.ex

```diff
@@ -0,0 +1,41 @@
+defmodule MongoosePushWeb.Endpoint do
+  use Phoenix.Endpoint, otp_app: :mongoose_push
+
+  # The session will be stored in the cookie and signed,
+  # this means its contents can be read but not tampered with.
+  # Set :encryption_salt if you would also like to encrypt it.
+  @session_options [
+    store: :cookie,
+    key: "_mongoose_push_key",
+    signing_salt: "URIBGSwL"
+  ]
+
+  # Serve at "/" the static files from "priv/static" directory.
+  #
+  # You should set gzip to true if you are running phx.digest
+  # when deploying your static files in production.
+  plug Plug.Static,
+    at: "/",
+    from: :mongoose_push,
+    gzip: false,
+    only: ~w(css fonts images js favicon.ico robots.txt)
+
+  # Code reloading can be explicitly enabled under the
+  # :code_reloader configuration of your endpoint.
+  if code_reloading? do
+    plug Phoenix.CodeReloader
+  end
+
+  plug Plug.RequestId
+  plug Plug.Telemetry, event_prefix: [:phoenix, :endpoint]
+
+  plug Plug.Parsers,
+    parsers: [:urlencoded, :multipart, :json],
+    pass: ["*/*"],
+    json_decoder: Phoenix.json_library()
```

Comment:
Do we know what is the default json encoder? I don't see it configured and I think it's better to use the one which is already used in the project. As far as I know MongoosePush already uses Jason. It can be configured by adding the following line:
```elixir
config :phoenix, :json_library, Jason
```
to `config/config.exs`